simple_history/admin.py

```diff
@@ -24,8 +24,10 @@
 
 USER_NATURAL_KEY = tuple(key.lower() for key in USER_NATURAL_KEY.split('.', 1))
 
+settings.SIMPLE_HISTORY_EDIT = getattr(settings, 'SIMPLE_HISTORY_EDIT', False)
```

Comment:
I don't think we want to be modifying settings here. Could we move it into a module constant?